polaris-react/src/components/TopBar/components/UserMenu/UserMenu.tsx

```diff
@@ -58,20 +60,22 @@ export function UserMenu({
         <Text as="p" alignment="start" fontWeight="medium" truncate>
           {name}
         </Text>
-        <Text
-          as="p"
-          variant="bodySm"
-          alignment="start"
-          color="subdued"
-          truncate
-        >
-          {detail}
-        </Text>
+        <span className={styles.Message}>
+          <Text
+            as="p"
+            variant="bodySm"
+            alignment="start"
+            color="subdued"
+            truncate
+          >
+            {detail}
+          </Text>
+        </span>
```

Comment:
To avoid adding extra elements outside of the beta flag, I would conditionally render the span

Reply:
It shouldn't have any visual impact, I was wondering if it's worth it.